importer/src/csv.ts

```diff
@@ -0,0 +1,56 @@
+import csv from 'csvtojson';
+import { Converter } from 'csvtojson/v2/Converter';
+import internal from 'stream';
+import { ParseOptions } from './index.js';
+
+export async function parseString(text: string, options: ParseOptions) {
+  return process(initConverter(options).fromString(text), options);
+}
+
+export async function parseFile(file: string, options: ParseOptions) {
+  return process(initConverter(options).fromFile(file), options);
```

Comment:
Do we handle different encodings? I wouldn't assume UTF-8 by default, and maybe make it configurable

Reply:
Good suggestion. I'm not sure if the csv parser already takes care of that somehow (checking if there's a BOM for UTF), but I guess for most cases if the encoding is a different one, we'd need a way to specify it.